tools/alive_parser.cpp

```diff
@@ -425,6 +481,8 @@ static Value& parse_operand(Type &type) {
     return get_fp_constant(yylval.fp_num, type);
   case NUM_STR:
     return get_num_constant(yylval.str, type);
+  case CSLT:
```

Comment:
this seems like we are going to have parsing problems with e.g. `<3, 4>`. This won't parse because the code assumes that `<number` is a type. I guess we would need more than 1 token of lookahead; not hard to implement, though.

Reply:
How about we assume `<number x` as a type in tokenizer?

Reply:
I guess that works as well :)